src/core/thread/mle.cpp

```diff
@@ -4162,7 +4162,7 @@ void Mle::LogError(MessageAction aAction, MessageType aType, Error aError)
 {
     if (aError != kErrorNone)
     {
-        if (aAction == kMessageReceive && (aError == kErrorDrop || aError == kErrorNoRoute))
+        if (aAction == kMessageReceive && (aError == kErrorDrop || aError == kErrorNoRoute || aError == kErrorAlready))
```

Comment:
This is fine. I am just curious about which sort of logs this is intended to suppress? Which MLE commands may generate `kErrorAlready`?

Reply:
Observed some logs which seems not so necessary to be in NOTE level
`2025-04-28 21:23:49.571  2089  2089 W otbr-agent: [W] Mle-----------: Failed to process Link Request: Already`

It turns out it might be not needed any more now, as Yakun has removed this error code for link request already 
https://github.com/openthread/openthread/pull/11030/files#diff-00af97322ba69de16d755e6948f5866d9c58fd877353b184b19d98b52879dbceL710

Reply:
Should we remove the `|| aError == kErrorAlready)`  then?

Reply:
yes, will remove in new patch